src/utils.js

```diff
@@ -9,6 +9,7 @@ import isEmpty from 'validator/lib/isEmpty';
 import isNumeric from 'validator/lib/isNumeric';
 
 import { EMAIL_TEMPLATE_FIELD_MAX_LIMIT, OFFER_ASSIGNMENT_EMAIL_SUBJECT_LIMIT } from '../src/data/constants/emailTemplate';
+import { EMAIL_ADDRESS_TEXT_FORM_DATA, EMAIL_ADDRESS_CSV_FORM_DATA } from '../src/data/constants/addUsers';
```

Comment:
nit: we're already in the `src` directory. these imports could be `from './data/constants/*`